### ISSUE TITLE
seqr Episign tag 

### ISSUE THREAD

**Issue by Glemiret** [broadinstitute/seqr#2803]
Add a tag next to a gene if an episignature is available either on a research basis (Dr Weksberg’s lab) or a clinical basis (Greenwood lab-Episign). The tag could be called “Episignature” and be positioned below the gene name and above the OMIM box with the other tags. You can use whatever color seems appropriate.

Here is the list of 76 genes where this tag should be added: 
KDM6A 
KMT2D 
NSD1 
HNRNPK 
KMT2A 
EZH2 
EED
SUZ12 
SRCAP 
CHD7 
KDM5C
ARID1B
DNMT3A 
EHMT1
DYRK1A 
CHD2
ASXL1 
ASXL3 
CREBBP
EP300
DNMT1
ADNP
ATRX 
CDK13
KMT2C
SMARCA2 
ANKRD11
HIST1H1E
KANSL1
KAT6A 
KAT6B 
CTCF
SETBP1
SETD5
BCL11A
CASK
DDX3X
USP9X
CNOT3
CTNNB1
HNRNPU
MED13L
POGZ
CHD3
BRWD3
HNRNPH2
KMT2B
KMT5B
NIPBL
PHF6
PQBP1
RAD21
SETD1B
SETD2
SMARCA4
SMARCB1
SMC1A
SMC3
SMS
SOX11
SRCAP
TET3
UBE2A
YARS2
YY1
ZBTB24
ZNF711
CHD8
DNMT3B
EZH2
FAM50A
HELLS
H1-4
KDM2B
ARID1A
CDCA7

**Comment by hanars** [comment]
In general, solutions that require engineering intervention on a regular basis as part of an expected workflow are not good solutions. A gene list currently shows a tag on any project it has been added to, but it wouldn't be too hard to make it a special gene list that every project gets by default. Making this a gene list that you or one of the analysts would create and then you would edit the way you would any other gene lists is much more maintainable than needing me to intervene to update the data. If displaying it like a gene list is not an option, however, we can look at other solutions

**Comment by Glemiret** [comment]
Chrissy and I created a public gene list named "episignature" and added it to RGP projects. The tag now appears in seqr (we tested one gene in a RGP case). Anne said to hold for now before adding this gene list to all other Broad CMG projects.

**Comment by lynnpais** [comment]
@Glemiret is the public "episignature" list working well? If a new tag is not needed, we can close out this ticket.

**Comment by Glemiret** [comment]
Yes we see a gene list tag added to a gene that is part of that gene list. Sometimes it is not an obvious tag to notice because there can be many other gene list tags of the same color with the gene, but I think this is the best solution. We can close the ticket, thanks!
